int32 marker_id
string waypoint
